java/src/jmri/jmrit/display/layoutEditor/TrackSegment.java

```diff
@@ -247,9 +247,8 @@ public boolean getBezier() {
     public void setBezier(boolean boo) {
         if (bezier != boo) {
             bezier = boo;
-            if (getBezier()) {
-                arc = false;
-                circle = false;
+            if (bezier) {
+                arc = circle = false;
```

Comment:
Please do not use this form. Per our [recommended practices](http://jmri.org/help/en/html/doc/Technical/RP.shtml), we follow the [Java Code Conventions](http://www.oracle.com/technetwork/java/codeconventions-150003.pdf) which specifically read 

> 10.4 Variable Assignments. "Avoid assigning several variables to the same value in a single statement. It is hard to read."